public/src/components/channelManagement/helpers/shared.ts

```diff
@@ -103,6 +106,7 @@ export const LIVEBLOG_EPIC_CONFIG: EpicEditorConfig = {
   allowVariantSeparateArticleCount: false,
   allowVariantTicker: false,
   allowVariantChoiceCards: false,
+  allowVariantSignInLink: true,
```

Comment:
It's probably worth setting the liveblog, apple news, and amp epic configs to `false` here. Those are all different components so we'll have to add the sign in link functionality separately there too. Whilst we don't have it, it's probs best to hide the option from users

Reply:
👍 Makes sense!